docs/design-system.md

```diff
@@ -0,0 +1,73 @@
+
+## What goes in the design system?
+
+The Design System is **only** for shared basics and components. In [atomic design terms](https://bradfrost.com/blog/post/atomic-web-design) we are sharing atoms, molecules and (some) organisms.  If it is a small pattern that’s repeated in a more complicated structure (like a menu that’s used in a dropDown, a search component and a popover menu) then that structure should exist in the design system. If it’s shared across both the editor and the dashboard, it should exist in the design system - like a button or a dialog. If it’s something complex only used by the editor, say the color picker, which uses a slider and some other smaller components in the design system, the color picker would live in the editor because it is not shared by the dashboard. Things that are not easily generalized should not live in the design system.
+
+## Theme
+
+With this update we get a full new color palette and theme. The structure is changing a little bit to try and keep things more organized. You can dig into the app [assets/src/design-system/theme](https://github.com/google/web-stories-wp/tree/main/assets/src/design-system) to see organization. Here’s brief overview of the organization to contextualize what you’re looking at.
+
+1. Colors, Typography, Borders, Breakpoints are in separate files and all added to the index’s `theme` that can be ingested at the root of each app. This file separation makes it easier to find theme values for developers who prefer to look in source code for the value they need.
+2. The new theme provides a dark and a light variant. The dark is the default. All colors are available regardless of variant, but the `theme.colors.bg` and `theme.colors.fg` are what you should gravitate towards using because the light and dark theme colors mirror each other. This will make dark and light mode functionality easier to handle down the road.
+3. Subdirectory called `helpers` - these are styles meant to be quick references for one off components in the editor and dashboard that aren’t in the design system.They’re also used in the design system for continuity.
+4. Subdirectory called `global` - these are exported as `ThemeGlobals` and should be added to app root code to handle global styles for both the dashboard and editor. Currently only the focus override exists here.
+5. Subdirectory called `constants` - these are constants that are used in both the dashboard and the editor (or needed also in the design system) so that we can have a single source of truth. You can import them via `THEME _CONSTANTS`. These include things like static widths of wordPress menus that all projects reference.
```

Comment:
```suggestion
5. Subdirectory called `constants` - these are constants that are used in the dashboard, the editor and across the  design system so that we can have a single source of truth. You can import them via `THEME _CONSTANTS`. These include things like static widths of wordPress menus that all projects reference.
```